tests/utils.py

```diff
@@ -73,5 +73,5 @@ def load_process_code(process_id):
 def get_defined_processes_from_files():
     return [
         os.path.splitext(os.path.basename(file_path))[0]
-        for file_path in glob.glob(f"../src/pg_to_evalscript/javascript_processes/*.js")
+        for file_path in glob.glob(f"../../src/pg_to_evalscript/javascript_processes/*.js")
```

Comment:
Why was this changed? Doesn't seem in scope for this PR. So that running tests from e.g. inside `integration_tests` directory works? In that case there should be a proper solution with absolute paths found. 
I assume this instruction from readme for running tests directly will no longer work:
https://github.com/openEOPlatform/openeo-pg-evalscript-converter#directly

Reply:
Yeah sorry, I messed something up. Deleted this change.